python/hail/dataset.py

```diff
@@ -1433,53 +1471,51 @@ def export_vcf(self, output, append_to_header=None, export_pp=False, parallel=Fa
 
         >>> hc.read('data/example.vds').export_vcf('data.example.vcf.bgz', export_pp=True, parallel=True)
```

Comment:
I'd remove the last two examples (too specialized) and make the example above for a block compressed vcf.